framework/include/ble_manager/ble_client.h

```diff
@@ -242,16 +241,17 @@ ble_result_e ble_client_operation_enable_notification(ble_client_operation_handl
  *   I/O operation with Server(Peripheral).
  * 
  * Input Parameters:
- *   handle  - This includes the handle ID and the attribute ID.
- *   data    - I/O buffer which trasfers data.
- *             (On read, this value should be allocated and set the length as the size of buffer.)
+ *   ctx          - The context of client.
+ *   attr_handle  - attribute handle.
+ *   data         - I/O buffer which trasfers data.
```

Comment:
Please change `trasfers` to `transfers`.

Reply:
I fixed it